aio_pika/robust_connection.py

```diff
@@ -72,13 +72,13 @@ def add_reconnect_callback(self, callback: Callable[[], None]):
         self._on_reconnect_callbacks.add(callback)
 
     async def connect(self, timeout=None):
-        if self.fail_fast:
-            return await super().connect(timeout=timeout)
-
         while True:
             try:
-                await super().connect(timeout=timeout)
-            except (ConnectionError, AMQPError):
+                return await super().connect(timeout=timeout)
+            except (ConnectionError, AMQPError) + PAMQP_EXCEPTIONS:
```

Comment:
Could you please define this tuple in global scope?

Reply:
Done.